clang-tools-extra/clangd/CodeComplete.cpp

```diff
@@ -2455,6 +2455,11 @@ bool isIncludeFile(llvm::StringRef Line) {
 }
 
 bool allowImplicitCompletion(llvm::StringRef Content, unsigned Offset) {
+  // Check if we're inside an empty template argument list
```

Comment:
nit: for consistency with the other checks in this function, could you:

 * move this after the "Look at last line before completion only" check; and
 * formulate the check using `ends_with`?